packages/windows/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "3.2.0"
+  changes:
+    - description: Add match_by_parent_instance option to windows perfmon datastream.
```

This changelog should be a user-facing message.

```suggestion
    - description: Expose the "Perfmon Match By Parent Instance" option in Windows Perfmon (default: true). When unchecked, unique instance names are used for processes with the same name.
```